Adaptors/MongoDB/src/IMongoQueryableExt.cs

```diff
@@ -40,4 +43,36 @@ internal static class IMongoQueryableExt
       }
     }
   }
+
+  public static IOrderedMongoQueryable<T> OrderByList<T>(this IMongoQueryable<T>            queryable,
+                                                         List<Expression<Func<T, object?>>> orderFields,
+                                                         bool                               ascOrder = true)
```

Comment:
Order should most likely be per "column", and not necessarily the same for all "columns".

Reply:
It is not needed and simplifies code for now